.github/workflows/integration-tests.yml

```diff
@@ -479,9 +480,13 @@ jobs:
       - name: Run Setup
         uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/setup-go@260e9b8a4c78d8c1fa6693a0e7144b40e41c7a35
         with:
-          test_download_vendor_packages_command: cd ./integration-tests && go mod download
+          test_download_vendor_packages_command: |
+            cd ./integration-tests
+            go mod download
+            # force download of test dependencies
+            go test -run=NonExistentTest ./smoke/... || echo "ignore expected test failure"
```

Comment:
How about adding a blank nonexistent test to avoid the failure msg?

Reply:
It is only needed in CI  for this specific caching instance so there is no need for the test to exist for any other reason. Whether this passes or fails also doesn't matter. The only thing that matters for this case is that the dependencies are downloaded.